github/github.go

```diff
@@ -986,6 +996,59 @@ func (g *GitHub) UpdateReleasePage(
 	return release, nil
 }
 
+// toRepositoryRelease builds a repository release from the set of options
+func (urpo *UpdateReleasePageOptions) toRepositoryRelease() *github.RepositoryRelease {
+	request := &github.RepositoryRelease{}
+
+	if urpo.Name != "" {
+		request.Name = &urpo.Name
+	}
+	if urpo.Body != "" {
+		request.Body = &urpo.Body
+	}
```

Comment:
Do we need those additional `!= ""` checks? Setting the pointer to an empty string could have the same effect.

Reply:
No longer in the PR because of the discussion going on in https://github.com/kubernetes-sigs/release-sdk/pull/288#discussion_r1429990159 and https://github.com/kubernetes-sigs/release-sdk/pull/288#discussion_r1429996068.